von_mises defines PI which doesn't allow RStan to compile

I get this error when compiling RStan:

```
rstan/inst/include/stansrc/stan/prob/distributions/univariate/continuous/von_mises.hpp:163:14: error: 
      expected unqualified-id
      double PI = stan::math::pi();
             ^
/Library/Frameworks/R.framework/Resources/include/R_ext/Constants.h:32:24: note: expanded from macro
      'PI'
#define PI             M_PI
                       ^
/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.9.sdk/usr/include/math.h:683:21: note: 
      expanded from macro 'M_PI'
#define M_PI        3.14159265358979323846264338327950288   /* pi             */
                    ^
1 error generated.
```

I think the solution is to fix Stan's code so:
      `double PI = stan::math::pi();`
is just defined `pi`.

--- a/src/stan/prob/distributions/univariate/continuous/von_mises.hpp
+++ b/src/stan/prob/distributions/univariate/continuous/von_mises.hpp
@@ -161,6 +161,7 @@ namespace stan {
       double U3 = uniform_rng(0.0, 1.0, rng) - 0.5;
       double sign = ((U3 >= 0) - (U3 <= 0));
 
+      //  it's really an fmod() with a positivity constraint
       return sign * std::acos(W) + fmod(fmod(mu,2*stan::math::pi())+2*stan::math::pi(),2*stan::math::pi());
     }
 

